AndroidAnnotations/androidannotations-core/androidannotations-test/src/main/java/org/androidannotations/test/ActivityWithServices.java

```diff
@@ -279,4 +279,28 @@
 	@SystemService
 	WindowManager windowManager;
 
+	AppWidgetManager methodInjectedAppWidgetManager;
```

Comment:
Never mind. The fields are packapege private because we have to access them from the test class.

I just said it because we want method injections to be able to inject to `private` fields (at least this is on of the reasons). But we still have to access the fields in the test class, so just forget my comment.